frontend/test/__support__/e2e/cypress.js

```
@@ -1,3 +1,5 @@
+import "@deploysentinel/cypress-debugger/support";
```

Have this on the very top so debugger catch all exceptions